docs/architecture/cloud-native/relational-vs-nosql-data.md

```diff
@@ -0,0 +1,275 @@
+
+High availability and massive scalability are often more critical to the business than strong consistency. Developers can implement techniques and patterns such as Sagas, CQRS, and asynchronous messaging to embrace eventual consistency.
+
+> Care must be taken with these descriptions as a new type of database, NewSQL, extend the relational database engine to support the horizontal scalability and scalable performance of NoSQL systems.
+
+## Considerations for Relational vs. NoSQL systems
+
+Based upon specific data requirements, a cloud-native-based microservice can implement a relational, NoSQL datastore or both.
+
+|  Consider a NoSQL datastore when: | Consider a relational database when: | 
+| :-------- | :-------- |
+| You have high volume workloads that require large scale | Your workload volume is consistent and requires medium to large scale |
+| Your workloads don't require ACID guarantees |  ACID guarantees are required |
+| Your data is dynamic and frequently changes | Your data is predictable and highly structured |
+| Data can be expressed without relationships | Data is best expressed relationally |  
+| You need fast writes and write safety isn't critical | Write safety is a requirement |  
+| Data retrieval is simple and tends to be flat | You work with complex queries and reports|
+| Your data requires a wide geographic distribution | Your users are more centralized | 
+| Your application will be deployed to commodity hardware, such as with public clouds | Your application will be deployed to large, high-end hardware | 
+|||
+
+In the next sections, we'll explore the options available in the Azure cloud for storing and managing your cloud-native data.
+
+## Database as a Service
+
+To start, you could provision an Azure virtual machine and install your database of choice for each service. While you'd have full control over the environment, you'd forgo many built-in features of the cloud platform. You'd also be responsible for managing the virtual machine and database for each service. This approach could quickly become time-consuming and expensive.
+
+Instead, cloud-native applications favor data services exposed as a [Database as a Service (DBaaS)](https://www.stratoscale.com/blog/dbaas/what-is-database-as-a-service/). Fully managed by a cloud vendor, these services provide built-in security, scalability, and monitoring. Instead of owning the service, you simply consume it as a [backing service](./definition.md#Request/Backing-services). The provider operates the resource at scale and bears the responsibility for performance and maintenance.
```

Comment:
bad anchor here

Reply:
Fixed, change from ./definition.md#Request/backing-services to /definition.md#backing-services